mm2src/coins/z_coin.rs

```diff
@@ -153,6 +153,10 @@ impl ZCoin {
 
     pub fn rpc_client(&self) -> &UtxoRpcClientEnum { &self.utxo_arc.rpc_client }
 
+    pub fn is_sapling_state_synced(&self) -> bool { self.z_fields.sapling_state_synced.load(AtomicOrdering::Relaxed) }
+
+    pub fn my_z_address_encoded(&self) -> String { self.z_fields.my_z_addr_encoded.clone() }
```

Comment:
Won't the compiler inline them itself? 🙂

Reply:
> Won't the compiler inline them itself? slightly_smiling_face

Not always. For example, compiler won't inline anything between the crates unless you specify it. Sometimes even in the same crate, the compiler won't inline it, it's not guaranteed so that's why they provided this attribute. I think it's even better to `inline` it with `always` option, but it's no needed for most cases. :slightly_smiling_face:

Reply:
Ok, thanks for the info 🙂 Will add `inline` in all such cases.